src/test/java/spoon/test/imports/ImportTest.java

```diff
@@ -403,8 +403,8 @@ public void testNestedAccessPathWithTypedParameter() throws Exception {
 		}
 		CtClass<?> mm = launcher.getFactory().Class().get("spoon.test.imports.testclasses2.AbstractMapBasedMultimap");
 		CtClass<?> mmwli = launcher.getFactory().Class().get("spoon.test.imports.testclasses2.AbstractMapBasedMultimap$WrappedList$WrappedListIterator");
-		assertTrue(mmwli.toString().indexOf("AbstractMapBasedMultimap<K, V>.WrappedList.WrappedIterator")>=0);
-		assertTrue(mm.toString().indexOf("AbstractMapBasedMultimap<K, V>.WrappedList.WrappedIterator")>=0);
+		assertTrue(mmwli.toString().indexOf("AbstractMapBasedMultimap<K, V>.WrappedCollection.WrappedIterator")>=0);
+		assertTrue(mm.toString().indexOf("AbstractMapBasedMultimap<K, V>.WrappedCollection.WrappedIterator")>=0);
```

Comment:
Ok I think there's a behaviour change here, but maybe it's better, I'll try to clarify it. 
What we had before was:
```
abstract class AbstractMapBasedMultimap<K, V> {
    private class WrappedCollection {
        class WrappedIterator {        }
    }

    private class WrappedList extends AbstractMapBasedMultimap<K, V>.WrappedCollection {
        private class WrappedListIterator extends AbstractMapBasedMultimap<K, V>.WrappedList.WrappedIterator {        }
    }
}
```

What we have now is:
```
abstract class AbstractMapBasedMultimap<K, V> {
    private class WrappedCollection {
        class WrappedIterator {        }
    }

    private class WrappedList extends AbstractMapBasedMultimap<K, V>.WrappedCollection {
        private class WrappedListIterator extends AbstractMapBasedMultimap<K, V>.WrappedCollection.WrappedIterator {        }
    }
}
```
which means if now you redefine class `WrappedIterator` in `WrappedList` it won't be take into account. 
But the original source code is:
```
abstract class AbstractMapBasedMultimap<K, V> {
	private class WrappedCollection {
		class WrappedIterator {
		}
	}
	private class WrappedList extends WrappedCollection {
		private class WrappedListIterator extends WrappedIterator {
		}
	}
}
```
And I don't think the WrappedIterator here point to WrappedList.WrappedIterator... WDYT @tdurieux ?